frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -66,6 +66,11 @@ self.addEventListener('message', function (event) {
   }
 })
 
+// Handle clicks on notifications issued via registration.showNotification().
+self.addEventListener('notificationclick', event => {
+  console.debug('[sw] Notification clicked:', event.notification)
+})
+
```

Comment:
This did get called in the SW console (not the main console) when I tested.

And this could indeed be used to focus the window (a.k.a Client) from which the `showNotification` call originated.
But when `path` is given in `makeNotification()` we also want to redirect to another page, and I'm not sure yet how to achieve that.

Reply:
We could probably leave this to @corrideat in his upcoming Push Notifications PR as he's building a communications system between the service worker and the open windows/tabs.